interface/forms/fee_sheet/review/fee_sheet_search_queries.php

```
@@ -27,7 +27,7 @@
  * @param string $search_query The text to search on.
  * @return array
  */
-function diagnosis_search($search_type_id, $search_type, $search_query)
+function diagnosis_search($search_type_id, $search_type, $search_query): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$search_type_id'. (undefined)

(UnusedFormalParameter)